### ISSUE TITLE
WCAG 1.4.3: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.bg-rose-500\/10)

### ISSUE THREAD

**Issue by azap026** [azap026/smetalabv3#85]
#### Issue
`Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Smetalab - Smart Engineering & Management](http://localhost:3000/app)
#### Element path
.bg-rose-500\/10
#### Snippet
`<span class="rounded-full px-2 py-1 text-xs font-semibold bg-rose-500/10 text-rose-600">Live</span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 4.05 (foreground color: #ec003f, background color: #ffe9ee, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 143.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
